src/ZXing.Net.Maui/ZXing.Net.MAUI/CameraBarcodeReaderViewHandler.cs

```diff
@@ -89,7 +89,7 @@ public void Focus(Point point)
 
 		public void AutoFocus()
 			=> cameraManager?.AutoFocus();
-
+#nullable enable
```

Comment:
Wasn't there any other way to handle null values instead of enabling nullable context?

Reply:
Maybe there is, but I couldn't find other solution at the time and went with approach that works. I don't think it's a big deal as semantics are not changed.